.github/workflows/fetch-lib.yml

```diff
@@ -0,0 +1,40 @@
+name: Update Dependencies (fetch-lib)
+
+on:
+  schedule:
+    - cron:  '0 10 * * 1'
```

Comment:
just asking: is there a way to put a limit on the number of retries? Like, if this action fails X times in a row and there have been no code changes, stop trying?

Asking mostly out of curiosity, because one such script sent me 20 emails over the weekend at some point.

Reply:
Also, should this not be in the master COS CI?